static/styles/lib/sidebar.scss

```diff
@@ -244,6 +244,20 @@
       cursor: pointer;
     }
   }
+
+	.instance-copy-right {
```

Comment:
BTW `.instance-copyright`? 🤓

Reply:
Hehe oops, luckily it gets removed anyways 😄